iceberg/openhouse/internalcatalog/src/main/java/com/linkedin/openhouse/internal/catalog/OpenHouseInternalTableOperations.java

```diff
@@ -607,4 +656,15 @@ private boolean isReplicatedTableCreate(Map<String, String> properties) {
                 CatalogConstants.OPENHOUSE_TABLE_VERSION, CatalogConstants.INITIAL_VERSION)
             .equals(CatalogConstants.INITIAL_VERSION);
   }
+
+  private List<String> getNewSchemasFromProps(TableMetadata metadata) {
```

Comment:
Do you think we can change the method name to say getIntermediateSchemasFromProps?